src/app/PCM/templates/tpl-sbe-row-add/sbe-add-boolean.js

```diff
@@ -0,0 +1,9 @@
+import { Box } from '@chakra-ui/react'
+
+const SBEAddBoolean = ({ askData, value, onChange, disabled }) => {
+  const questionCode = askData?.questionCode || ''
+
+  return <Box />
```

Comment:
why is this component returning only Box. Is return logic left to be coded later on?

Reply:
Yes, I probably should have omitted it from this PR